bindings/nodejs/Cargo.toml

```diff
@@ -1,32 +1,35 @@
 [package]
-name = "wallet"
+name = "iota-wallet-nodejs"
```

Comment:
We can't just change it because of `The crate name will be read from the npm_package_name environment variable. If the package name includes a namespace (@namespace/package), the namespace will be removed when matching the crate name (package).` and our npm package is `@iota/wallet`
https://github.com/neon-bindings/cargo-cp-artifact